TSE Review.

carlosveigadev opened the pull request:
Added all the elements and style for the Page applying all the theories learned about the position and floating elements. Including float, box model, flexbox, CSS reset. We also made the recommended validations with Linter and W3 Validator.

**Navigation Bar**

- Defined the fixed position
- Included Buttons, Logo, and Links

**First Article, Second Article and Third Article.**

- Included Media Icons
- Added the paragraphs
- Embedded images and video
- Position the section and its elements
- Style the text

**Explanation Grid**

- Added the images
- Added the paragraphs

**Asides**

- Position the aside using float
- Position of the Elements using flex
- Added the paragraphs

**Footer**

- Styled and positioned the footer list 
- Positioned all the list with its links
- Included the borders

epinczinger committed:
start nav bar and header

carlosveigadev committed:
we finished styling the main figure and added some of the paragraphs

epinczinger committed:
add p, img, and flex grid

carlosveigadev committed:
we fixed some stylings in the explanation images, along with the styling of anchors and the main section of the articles.

epinczinger committed:
finished paragraphs and starting grid and footer

epinczinger committed:
images/footes styling

carlosveigadev committed:
finished styling editors pick and added the footer.

epinczinger committed:
fixed html errors with validator

carlosveigadev committed:
added stickler files

carlosveigadev committed:
fixed stylint errors and added some fonts

epinczinger committed:
update readme

epinczinger committed:
fixed lint errors and added screenshot

epinczinger committed:
update screenshot file

epinczinger committed:
fixing errors

epinczinger committed:
fixed spacing erros

carlosveigadev committed:
Added some comments in the .html file.

epinczinger committed:
delete .gitignore

carlosveigadev committed:
fixed the image on the footer

epinczinger committed:
fix live link

carlosveigadev committed:
removed / from image path on the footer.

carlosveigadev committed:
removed / from image of cosmos html code

carlosveigadev committed:
removed / inflation theory section

carlosveigadev committed:
Merge branch 'master' into main_page

lucasmazo32 commented on begin.css:
```diff
@@ -0,0 +1,482 @@
+/* General */
```
- [x] Elements should be aligned with flex, grid, and float. For this project you only use flex and float. Please align items with grid. 
> ![image](https://user-images.githubusercontent.com/57414789/82390783-3ef55180-9a05-11ea-97f3-c4d5b9862e70.png)

carlosveigadev replied:
This requirement was done as asked.

lucasmazo32 commented on begin.html:
```diff
@@ -0,0 +1,792 @@
+          <li><a href="#">Subscriptions</a></li>
+        </ul>
+      </div>
+    </footer>
```
- [x] There is a part of the page that sticks to the bottom. Please add it.

> ![image](https://user-images.githubusercontent.com/57414789/82390898-8f6caf00-9a05-11ea-8e22-9f846a14cfbd.png)

carlosveigadev replied:
This requirement was done as asked.

lucasmazo32 reviewed (changes requested):
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

epinczinger committed:
add sticky footer

carlosveigadev committed:
added the bottom fixed element

epinczinger committed:
fix issues and tse requested changes

carlosveigadev committed:
resolve issues from may 20th, changed name of .html and moved the css files to /assets

carlosveigadev committed:
fixed background image not working properly

epinczinger committed:
inflation image fixed

carlosveigadev committed:
fixed some linter using npx

carlosveigadev commented:
@lucasmazo32 

- We added the grid element to the inflation explanation;
- We added the bottom fixed menu;
- We also added the suggestions that our team gave us to improve the project.

trekab reviewed (approved):
# Approved!
Hi @wrakc and @epinczinger, you have done a great job! :clap: You may merge to master, submit the project completion form, and continue to your next project.

_Happy coding!_ :wink:
__Treasure Kabareebe__
